Revisar posible error en el Outline

En Windows está ocurriendo

```
Caused by: java.lang.ClassCastException: org.eclipse.ui.part.MessagePage cannot be cast to org.eclipse.xtext.ui.editor.outline.impl.OutlinePage
	at org.uqbar.project.wollok.ui.WollokActivator$4.run(WollokActivator.java:174)
	at org.eclipse.ui.internal.UILockListener.doPendingWork(UILockListener.java:162)
	at org.eclipse.ui.internal.UISynchronizer$3.run(UISynchronizer.java:154)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:35)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:182)
```

--- a/org.uqbar.project.wollok.ui/src/org/uqbar/project/wollok/ui/WollokActivator.xtend
+++ b/org.uqbar.project.wollok.ui/src/org/uqbar/project/wollok/ui/WollokActivator.xtend
@@ -111,7 +111,11 @@ class WollokActivator extends org.uqbar.project.wollok.ui.internal.WollokActivat
 		Display.^default.syncExec [
 			val outlineView = activePage.findView("org.eclipse.ui.views.ContentOutline") as ContentOutline
 			if (outlineView !== null) {
-				(outlineView.currentPage as OutlinePage).scheduleRefresh
+				val outlinePage = outlineView.currentPage
+				try {
+					(outlinePage as OutlinePage).scheduleRefresh
+				} catch (ClassCastException e) {
+				}
 			}
 		]
 	}

